modules/ossm-integrating-distr-tracing-tempo.adoc

```diff
@@ -0,0 +1,90 @@
+////
+This module is included in the following assemblies:
+* service_mesh/v2x/ossm-observability.adoc
+////
+
+:_mod-docs-content-type: PROCEDURE
+[id="ossm-integrating-distr-tracing-tempo_{context}"]
+= Integrating distributed tracing platform (Tempo)
+//this one is specific to Tempo
+//another module may be specific to OTEL
+//another module may be specific to Kiali
+//they should all reside in ossm-observability, and in the order they need to happen/be completed
+
+You can expose tracing data to the {DTProductName} (Tempo) stack by appending a named element and `zipkin` provider to the `spec.meshConfig.extensionProviders` specification. Then, a telemetry custom resource configures Istio proxies to collect trace spans and send them to the Tempo distributor service endpoint.
+
+.Prerequisites
+
+Install the {DTProductName} (Tempo) stack by following the instructions provided in 
+ifndef::openshift-rosa,openshift-dedicated[]
+xref:../../distr_tracing/distr_tracing_tempo/distr-tracing-tempo-installing.adoc[Installing the distributed tracing platform (Tempo)].
+endif::[]
+
+.Procedure
+
+. Configure the `ServiceMeshControlPlane` resource to connect to the Tempo Stack instance in the `tracing-system` namespace:
++
```

Comment:
Is there a command to access this since it is in the `tracing-system` NS? Usually when altering a YAML file I say that there should be an accompanying command.